.github/workflows/ci.yml

```diff
@@ -21,8 +21,12 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 16.14.2
+      - name: Enable Corepack
+        run: corepack enable
       - name: Setup yarn
-        run: npm install -g yarn@1.22.15
+        run: corepack prepare yarn@1.22.15 --activate
+      - name: Setup pnpm
+        run: corepack prepare pnpm@7.0.1 --activate
```

Comment:
Managing package manager installations inside and outside Corepack do not mix well, so we got to do it with Corepack only.